source/tutorials/redis.rst

```diff
@@ -15,74 +15,65 @@ infrastructure development.
 Description
 ***********
 
-Redis is an in-memory key:value store designed for quick lookups, accessible over a network. While the `redis data structure store`_ can serve
+Redis is an in-memory key:value store designed for quick lookups, accessible 
+over a network. While the `redis data structure store`_ can serve
 as a NoSQL database for a web application, it's also easy to integrate into an
 existing stack. For example, you could use the Redis caching layer for
 real-time responses on a leaderboard in a gaming app. Redis offers many client
 libraries with language-specific bindings for Python\*, Perl\*, Ruby, and more.
 
-Prerequisites
-*************
-
-* Install the :command:`redis-native` bundle in |CL|
-* Install the :command:`containers-basic` bundle in |CL| (only required in
-  Example 2)
-
 Install the Redis bundle
 ************************
 
-In |CL|, find Redis in the :command:`redis-native` bundle.
+#. Log in as a user with administrative privilege.
 
-#. Open a terminal and login as an administrative user.
+#. Open a terminal.
```

Comment:
I recommend adding a swupd update here, before installing the bundle. 

I was testing on an older version (~32890), and for some reason, running `sudo systemctl enable --now redis`
just hung on creating a symlink to the proper `/etc` directory.

Let's eliminate this possibility by running a `sudo swupd update`

Reply:
Did running "swupd update" fix your issue?

Reply:
Yes, it did. That's why I'm recommending it.

Reply:
Error occurred on GCE VM instance and on my physical NUC. It won't hurt to add this command.